ghauth.js

```diff
@@ -5,13 +5,19 @@ const read = promisify(require('read'))
 const hyperquest = require('hyperquest')
 const bl = require('bl')
 const appCfg = require('application-config')
+const querystring = require('querystring')
+const ora = require('ora')
+const logSymbols = require('log-symbols')
+const { Octokit } = require('@octokit/rest')
 
 const defaultUA = 'Magic Node.js application that does magic things with ghauth'
 const defaultScopes = []
-const defaultNote = 'Node.js command-line app with ghauth'
-const defaultAuthUrl = 'https://api.github.com/authorizations'
+const defaultDeviceCodeUrl = 'https://github.com/login/device/code'
+const defaultDeviceAuthUrl = 'https://github.com/login/device'
+const defaultAccessTokenUrl = 'https://github.com/login/oauth/access_token'
+const defaultOauthAppsBaseUrl = 'https://github.com/settings/connections/applications/'
+const defaultPatUrl = 'https://github.com/settings/tokens'
```

Comment:
Thinking of accepting a base url, and building all of these dynamically, that way callers only have to customize one base url for GH enterprise.